blockbase/sass/blocks/_post-comments.scss

```diff
@@ -1,3 +1,48 @@
+body:not(.highlander-enabled) {
+	.wp-block-post-comments {
+		form {
+			display: grid;
```

Comment:
I think we only need to cover this line, but I don't think it matters much that we added the rest so I'm not too bothered by it

Reply:
I prefered keeping all the grid CSS together for readability.

Reply:
makes sense, works for me!